app/src/main/java/com/blackboxteam/app/blackbox/LoginScreenActivity.java

```diff
@@ -0,0 +1,82 @@
+package com.blackboxteam.app.blackbox;
+
+import android.content.Intent;
+import android.support.design.widget.Snackbar;
+import android.support.design.widget.TextInputLayout;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.blackboxteam.app.blackbox.util.Validation;
+
+import static com.blackboxteam.app.blackbox.util.Validation.validateEmail;
+import static com.blackboxteam.app.blackbox.util.Validation.validateMinLength;
+
+public class LoginScreenActivity extends AppCompatActivity implements View.OnClickListener {
+
+    Button loginBtn;
+    EditText etEmail, etPassword;
+    TextInputLayout tiEmail, tiPassword;
+    Validation validate;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login_screen);
+
+        loginBtn = (Button) findViewById(R.id.btn_login);
+        etEmail = (EditText) findViewById(R.id.et_email);
+        etPassword = (EditText) findViewById(R.id.et_password);
+        tiEmail = (TextInputLayout) findViewById(R.id.ti_email);
+        tiPassword = (TextInputLayout) findViewById(R.id.ti_password);
+        loginBtn.setOnClickListener(this);
+    }
+
+    private void login() {
+        setError();
+        String email = etEmail.getText().toString();
+        String password = etPassword.getText().toString();
+
+        int err = 0;
+
+        if (!validateEmail(email)) {
+            err++;
+            tiEmail.setError("Email should be valid");
+        }
+
+        if (!validateMinLength(password, 6)) {
```

Comment:
I think the literal `6` would be better as a constant. In case you want to change it, there is only one place where you have to update it and you won't have to scan the code. What do you think?